packages/adapter-netlify/index.js

```diff
@@ -51,7 +51,7 @@ export default function ({ split = false } = {}) {
 			const redirects = [];
 
 			const replace = {
-				APP: './server/app.js'
+				'0APP': './server/app.js' // digit prefix prevents CJS build from using this as a variable name
```

Comment:
i might add a bit more detail
```suggestion
				'0APP': './server/app.js' // digit prefix prevents CJS build from using this as a variable name to ensure replace doesn't replace occurrences of variables with this string
```